### ISSUE TITLE
[BUG]: Bug or feature?

### ISSUE THREAD

**Issue by MaximKurbanov** [Unity-Technologies/com.unity.webrtc#972]
### Package version

3.0.0-pre.6

### Environment

```markdown
* OS: All
* Unity version: 2021+
```

### Steps To Reproduce

By default, when sending and receiving video, the picture flips vertically.

Sender:
//Flip vertically
`Graphics.Blit(sourceTexture_, destTexture_, s_verticalScale, s_verticalOffset);`
Receiver:
and receiver flip back
`m_renderer = new UnityVideoRenderer(this, needFlip: true);`

### **It is not possible for the plugin user to change the auto flip of the receiver**
```
/// <summary>
/// Video Receiver
/// </summary>
/// <param name="ptr"></param>
internal VideoStreamTrack(IntPtr ptr)
    : base(CreateVideoTrack(ptr))
{
    if (!s_tracks.TryAdd(self, new WeakReference<VideoStreamTrack>(this)))
        throw new InvalidOperationException();

    m_dataptr = Marshal.AllocHGlobal(Marshal.SizeOf(typeof(VideoStreamTrackData)));
    Marshal.StructureToPtr(m_data, m_dataptr, false);

    m_renderer = new UnityVideoRenderer(this, true); <---
}
```

**Comment by karasusan** [comment]
@MaximKurbanov 


VideoStreamTrack provides a constructor with a `needFlip` argument. Can you try this?

```
public VideoStreamTrack(Texture texture, bool needFlip = true)
```

**Comment by MaximKurbanov** [comment]
@karasusan 
This constructor for sender!

**Comment by MaximKurbanov** [comment]
Fixed #971
